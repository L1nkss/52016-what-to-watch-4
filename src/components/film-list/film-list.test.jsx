import FilmList from "./film-list";

const films = [
  {
    name: `Bohemian Rhapsody`,
    image: `bohemian-rhapsody`
  },
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    image: `fantastic-beasts-the-crimes-of-grindelwald`
  },
  {
    name: `Macbeth`,
    image: `macbeth`
  },
  {
    name: `Aviator`,
    image: `aviator`
  },
  {
    name: `We need to talk about Kevin`,
    image: `we-need-to-talk-about-kevin`
  },
  {
    name: `What We Do in the Shadows`,
    image: `what-we-do-in-the-shadows`
  },
  {
    name: `Johnny English`,
    image: `johnny-english`
  },
  {
    name: `Shutter Island`,
    image: `shutter-island`
  },
];

describe(`Testing FilmList component`, () => {
  it(`Component should successfully rendered`, () => {
    const component = renderer.create(<FilmList films={films} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
