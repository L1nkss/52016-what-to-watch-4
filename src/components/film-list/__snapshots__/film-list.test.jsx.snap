// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing FilmList component Component should successfully rendered 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
      onClick={[Function]}
    >
      <video
        className="player__video"
        muted={true}
        poster="img/bohemian-rhapsody.jpg"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
  >
    <div
      className="small-movie-card__image"
      onClick={[Function]}
    >
      <video
        className="player__video"
        muted={true}
        poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="movie-page.html"
        onClick={[Function]}
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
</div>
`;
